### DISCUSSION TITLE
Welcome to tlc-remaster Discussions!

### DISCUSSION THREAD

**Discussion by Aeneas137** [Starflight-The-Lost-Colony/tlc-remaster#36] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by acoliver** [comment]
Hi. Are any of you still around? I just upgraded the build and made it multiplatform. I'm about to add a release process. Would you be interested in that as a PR or should I maintain it external?

**Reply by Aeneas137** [reply]
it was originally cross-platform and it was a pain in the ass so for one of the past releases I removed code that just got in the way of new features and bug fixes. I am a fan of Linux but this code base was messy due to too many fingers in the cookie jar. it wasn't worth the trouble at the time. and today you can just use wine or add it as an external game in steam. This is the one and only code base for TLC. You're free to play with the sources but I wouldn't want to try to integrate any commits. if you've followed any of the discussions on discord, everyone agreed that this old code base needed to be retired. I'm working on a retro version, porting it to old consoles and computers.

**Reply by Aeneas137** [reply]
but feel free to play with the sources. they've been used in a couple of other projects like the Genesis remake.

**Reply by acoliver** [reply]
ok I'll keep it forked then. Thanks for the reply.

**Comment by acoliver** [comment]
should anyone want to play on Mac here: https://github.com/acoliver/tlc-remaster/releases - the windows doesn't seem to work atm but I don't really have a windows box to test on, could be the virtual machine env. Didn't test on Linux but ItShouldWork(tm).

https://github.com/acoliver/tlc-remaster/releases
